packages/create-bison-app/template/package.json.ejs

```diff
@@ -41,7 +41,7 @@
     "test:e2e": "cross-env DOTENV_CONFIG_PATH=.env.test ts-node -r dotenv/config $(yarn bin)/start-server-and-test 'yarn test:server' http://localhost:3001 'yarn cypress:run'",
     "test:e2e:local": "cross-env CYPRESS_LOCAL=true CYPRESS_BASE_URL=http://localhost:3000 cypress open",
     "test:server": "next start --port 3001",
-    "ts-node": "ts-node-dev --project tsconfig.cjs.json",
+    "ts-node": "ts-node-dev --project tsconfig.cjs.json -r tsconfig-paths/register",
```

Comment:
This is what makes `ts-node` work with the path alias. Without it, none of the scripts run with `ts-node` would be able to use path aliases.